src/IniFile.cpp

```diff
@@ -49,6 +49,9 @@ cIniFile::cIniFile(void) :
 
 bool cIniFile::ReadFile(const AString & a_FileName, bool a_AllowExampleRedirect)
 {
+
+	m_Filename = a_FileName;
+
```

Comment:
I specifically removed the filename storage from the class, so that each write operation needs to specify what filename to write to. Why are you adding it back in?